pkg/controller/pxcrestore/controller.go

```diff
@@ -233,6 +235,30 @@ func (r *ReconcilePerconaXtraDBClusterRestore) reconcileStateStartCluster(ctx co
 	return rr, nil
 }
 
+func validate(ctx context.Context, restorer Restorer, cr *api.PerconaXtraDBClusterRestore) error {
+	job, err := restorer.Job()
+	if err != nil {
+		return errors.Wrap(err, "failed to create restore job")
+	}
+	if err := restorer.ValidateJob(ctx, job); err != nil {
+		return errors.Wrap(err, "failed to validate job")
```

Comment:
this error message is the same with the validation for the pitr, maybe we should keep them separate.